system/datatypes/peer_reviewed_article.php

```diff
@@ -1462,6 +1463,17 @@ protected function createCommentDomElement(HyphaDomElement $comment, $review = f
 			return '<span><a href="' . $url . '" target="_blank">' . $url . '</a>' . $suffix.'</span>';
 		}, $commentHtml);
 		$commentHtml = nl2br($commentHtml);
+		if(strlen($commentHtml) > self::MAX_COMMENT_LENGTH){
```

Comment:
This limits to 100chars, while @dianawi suggested cutting on 100 *words*. Also, cutting on whitespace only probably results in a nicer-to-read result as well, and might solve an issue with cutting links halfway (see below).

Reply:
Thanks! Switched to 100 * 6 chars which, on average, comes down to roughly 100 words.

Reply:
Sounds pragmatic :-)